tensorflow/core/api_def/base_api/api_def_Acosh.pbtxt

```diff
@@ -1,4 +1,11 @@
 op {
   graph_op_name: "Acosh"
   summary: "Computes inverse hyperbolic cosine of x element-wise."
+  description: <<END
+Given an input tensor, the function computes inverse hyperbolic cosine of every element.
+Input range is [1, infinity). It returns `nan` if the input lies outside the range.
```

Comment:
I'd say inf instead of infinity

Please put backticks around `[1, inf)` so it renders as code.